### PR TITLE
20220418-1118

### PR DESCRIPTION
**Pull request by UTCGilligan**
Needs style.css and Readme changes.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -5,7 +5,7 @@
  Author:       Bridget Hornsby
  Author URI:   https://www.utc.edu/
  Template:     genesis
- Version:      1.5.2
+ Version:      1.4.3
```
Comment (UTCGilligan) [inline style.css:8]: Wrong version number.

File: README.md [hunk 2]
```diff
@@ -10,14 +10,12 @@ Author: Bridget Hornsby
```
Comment (UTCGilligan) [inline README.md:10]: The theme URI is misspelled. It does not show here, but it has uct instead of utc in the URI.

Comment (UTCbridget) [reply README.md:10]: Doesn't show on my local either. Though the versioning does.

Comment (UTCbridget) [reply README.md:10]: This is corrected and merged into develop: https://github.com/UTCWeb/utc-tailwind-genesis-theme/pull/22

**Review by UTCbridget — COMMENTED** [review]
These files look good to me. README.md and styles.css here says v 1.5.3(?) which is correct.
And I don't see the URI misspelling in either file.

**Comment by UTCGilligan** [comment]
mispelling in the URI in Readme is not in the link, it's in the text of the link, "uct" vs "utc".

**Pull request closed without merging**